JitsiConference.js

```diff
@@ -42,6 +42,7 @@ import VideoType from './service/RTC/VideoType';
 import VideoSIPGW from './modules/videosipgw/VideoSIPGW';
 import * as VideoSIPGWConstants from './modules/videosipgw/VideoSIPGWConstants';
 import * as XMPPEvents from './service/xmpp/XMPPEvents';
+import JITSI_MEET_MUC_TOPIC from './modules/xmpp/ChatRoom';
```

Comment:
Can you confirm for me that this imports the correct value? I would assume you needed to do `import { JITSI_MEET_MUC_TOPIC } from './modules/xmpp/ChatRoom'`.

Reply:
Yes, fixed! I have no idea how this used to work...